shared-bindings/aesio/aes.c

```diff
@@ -98,33 +98,49 @@ STATIC mp_obj_t aesio_aes_make_new(const mp_obj_type_t *type, size_t n_args,
     return MP_OBJ_FROM_PTR(self);
 }
 
-STATIC mp_obj_t aesio_aes_rekey(size_t n_args, const mp_obj_t *pos_args) {
+//|     def rekey(
+//|         self,
+//|         key: ReadableBuffer,
+//|         iv: Optional[ReadableBuffer] = None,
```

Comment:
```suggestion
//|         IV: Optional[ReadableBuffer] = None,
```

Reply:
What capitalization is correct here? It looks like CP will parse `IV`.

Reply:
You are right. Will fix that. Thank you!